Sources/NIO/ByteBuffer-int.swift

```diff
@@ -97,22 +97,10 @@ extension ByteBuffer {
 extension UInt64 {
     /// Returns the next power of two.
     public func nextPowerOf2() -> UInt64 {
-        guard self > 0 else {
+        if self == 0 {
```

Comment:
I don't think we should take this change. Happy to make it `guard self != 0 else { ... }` though. But the `guard` tells/should tell the compiler that this is the less likely case and it can optimise for that.